splitio/sync/impression.py

```diff
@@ -79,11 +79,15 @@ def __init__(self, impressions_api, impressions_manager):
 
         """
         self._impressions_api = impressions_api
-        self._impressions_manager = impressions_manager
+        self._impressions_counter = imp_counter
 
     def synchronize_counters(self):
         """Send impressions from both the failed and new queues."""
-        to_send = self._impressions_manager.get_counts()
+
+        if not isinstance(self._impressions_counter, Counter):
```

Comment:
?
does this accept soemthing other than a counter?

Reply:
only counter, it is a check if the instance is not none.